packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/project/data/versioning/dto/IRestCommitReference.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.project.data.versioning.dto;
+
+import java.text.DateFormat;
+
+import org.eclipse.sirius.web.application.dto.IRestRecord;
+
+/**
+ * Interface for the REST CommitReference DTO.
+ *
+ * @author arichard
+ */
+public interface IRestCommitReference extends IRestRecord {
+
+    DateFormat created();
```

Comment:
Please prefer `Instant`, `DateFormat` is not a date, it's a java.text utility class to help formatting dates. We can see how the instant will be formatted later.

Reply:
Yes, sorry I missed that.